samples/application_development/external_lib/README.rst

```diff
@@ -9,3 +9,12 @@ A simple example that demonstrates how to include an external static library
 into the Zephyr build system.
 The demonstrates both how to build the external library using a different build
 system and how to include the built static library.
+
+Windows Note
+************
+
+To use this sample on a Windows host operating system, GNU Make needs to be in
+your path. The pre-built application can be downloaded from
```

Comment:
Please also mention that you can just do:
`choco install make`
for Chocolatey users on Windows. Since that is the package manager we recommend in the GSG, I think this should come first in the `.rst`.

Reply:
Fixed